app/views/components/blockEditor.html

```diff
@@ -35,6 +35,7 @@
     <gender-question block="block" ng-switch-when="genderQuestion"></gender-question>
     <date-question block="block" ng-switch-when="dateQuestion"></date-question>
     <year-in-school-question block="block" ng-switch-when="yearInSchoolQuestion"></year-in-school-question>
+    <birth-date-question block="block" ng-switch-when="birthDateQuestion"></birth-date-question>
```

Comment:
I might even just reuse  the `date-question` component here. It can still have a type of `birthDateQuestion` but just renders a date question since there's no logic in either component (directive) and the templates are the same.